app/models/requirement.rb

```diff
@@ -86,6 +88,38 @@ def has_data_validation?
     false
   end
 
+  def count_of_jurisdictions_using
+    JurisdictionTemplateVersionCustomization
+      .joins(:template_version)
+      .where(template_versions: { status: "published" })
+      .where(
+        "customizations -> 'requirement_block_changes' -> :requirement_id -> 'enabled_elective_field_ids' @> :id",
+        requirement_id: id.to_s,
+        id: "[\"#{id}\"]",
+      )
+      .count
+  end
+
+  def count_by_reason(reason)
+    return 0 unless JurisdictionTemplateVersionCustomization::ACCEPTED_ENABLED_ELECTIVE_FIELD_REASONS.include?(reason)
+
+    JurisdictionTemplateVersionCustomization
+      .joins(:template_version)
+      .where(template_versions: { status: "published" })
```

Comment:
I think you have to chain the following(same as `count_of_jurisdictions_using` ) before the `select`
``` .where(
        "customizations -> 'requirement_block_changes' -> :requirement_id -> 'enabled_elective_field_ids' @> :id",
        requirement_id: id.to_s,
        id: "[\"#{id}\"]",
      )
 ```

Otherwise this is checking the count for the reason for all elective fields, not for this specific requirement